core/platform/email/mailgun_email_services.py

```diff
@@ -16,10 +16,9 @@
 
 """Provides mailgun api to send email."""
 
-import requests
-
 from core.platform.email import gae_email_services
 import feconf
+import requests
```

Comment:
requests is 3rd-party. There are other third-party libs I'm seeing too that you haven't detected, so, again, please check this whole PR carefully.

Reply:
Okay, sure!

Reply:
@seanlip: I have added requests to the known third-party list. I will try to add others. Can you please list others which have been missed?  
Thanks!

Reply:
I think I mentioned before how to check this, but maybe I wasn't clear enough. Stuff like `import feconf` and `import utils` is fine because, at the top level of the codebase, there are feconf.py and utils.py. But `import cloudstorage` isn't fine because there's no cloudstorage.py at the top-level of the codebase, so that's a third-party lib. That's how you can check.

Reply:
Thank you @seanlip! I will see this. I have included all the libraries which were being installed by the the bash scripts for now. Thanks!